code.py

```diff
@@ -71,6 +71,7 @@ def check_for_scheduled_app_switch(self):
         hour, minute, second = now.tm_hour, now.tm_min, now.tm_sec
         # At midnight change to clock
         if hour == 0 and minute == 0 and second == 0 and not isinstance(self.current_app, ClockApp):
+            self.real_time.sync_time_online()
```

Comment:
this shouldn't be necessary, because we are already checking inside the main while loop whether it's been more than 5 hours since the last clock sync: `self.real_time.check_for_time_sync()`.

Maybe we need to reduce from 5 back to 2 hours in between the syncs?

Reply:
I guess putting the sync inside this if will cover for DST changes 🥇

Reply:
Not sure if I follow :) do you think we need sync here or not? 🙈